src/modules/Footer/Menu/Links.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+type LinkLabel = "About Us" | "Our Work" | "Client" | "Our Blog" | "Contact US";
+
+const links: Array<{ label: LinkLabel; path: string }> = [
+  { label: "About Us", path: "#About-section" },
+  { label: "Our Work", path: "#work-section" },
+  { label: "Client", path: "#client-section" },
+  { label: "Our Blog", path: "#blog-section" },
+  { label: "Contact US", path: "#contact-section" },
+];
+
+const Menu = () => {
+  const [active, setActive] = useState("");
```

Comment:
Why you need "active" state?

I don't like that you are doing copy-paste without thinking what you are taking.